2025-05-29 21:08:40,350 INFO    StreamThr :2864158 [internal.py:wandb_internal():85] W&B internal server running at pid: 2864158, started at: 2025-05-29 21:08:40.349635
2025-05-29 21:08:40,351 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: status
2025-05-29 21:08:40,353 INFO    WriterThread:2864158 [datastore.py:open_for_write():87] open: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/run-xb0u8kog.wandb
2025-05-29 21:08:40,353 DEBUG   SenderThread:2864158 [sender.py:send():379] send: header
2025-05-29 21:08:40,356 DEBUG   SenderThread:2864158 [sender.py:send():379] send: run
2025-05-29 21:08:40,745 INFO    SenderThread:2864158 [dir_watcher.py:__init__():211] watching files in: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files
2025-05-29 21:08:40,745 INFO    SenderThread:2864158 [sender.py:_start_run_threads():1188] run started: xb0u8kog with start time 1748552920.348772
2025-05-29 21:08:40,749 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: check_version
2025-05-29 21:08:40,749 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: check_version
2025-05-29 21:08:40,827 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: run_start
2025-05-29 21:08:40,834 DEBUG   HandlerThread:2864158 [system_info.py:__init__():26] System info init
2025-05-29 21:08:40,834 DEBUG   HandlerThread:2864158 [system_info.py:__init__():41] System info init done
2025-05-29 21:08:40,834 INFO    HandlerThread:2864158 [system_monitor.py:start():194] Starting system monitor
2025-05-29 21:08:40,834 INFO    SystemMonitor:2864158 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-05-29 21:08:40,835 INFO    HandlerThread:2864158 [system_monitor.py:probe():214] Collecting system info
2025-05-29 21:08:40,835 INFO    SystemMonitor:2864158 [interfaces.py:start():188] Started cpu monitoring
2025-05-29 21:08:40,836 INFO    SystemMonitor:2864158 [interfaces.py:start():188] Started disk monitoring
2025-05-29 21:08:40,836 INFO    SystemMonitor:2864158 [interfaces.py:start():188] Started gpu monitoring
2025-05-29 21:08:40,836 INFO    SystemMonitor:2864158 [interfaces.py:start():188] Started memory monitoring
2025-05-29 21:08:40,836 INFO    SystemMonitor:2864158 [interfaces.py:start():188] Started network monitoring
2025-05-29 21:08:40,844 DEBUG   HandlerThread:2864158 [system_info.py:probe():152] Probing system
2025-05-29 21:08:40,845 DEBUG   HandlerThread:2864158 [system_info.py:_probe_git():137] Probing git
2025-05-29 21:08:40,849 DEBUG   HandlerThread:2864158 [system_info.py:_probe_git():145] Probing git done
2025-05-29 21:08:40,849 DEBUG   HandlerThread:2864158 [system_info.py:probe():200] Probing system done
2025-05-29 21:08:40,849 DEBUG   HandlerThread:2864158 [system_monitor.py:probe():223] {'os': 'Linux-6.8.0-55-generic-x86_64-with-glibc2.39', 'python': '3.12.2', 'heartbeatAt': '2025-05-29T21:08:40.844573', 'startedAt': '2025-05-29T21:08:40.346089', 'docker': None, 'cuda': '9.0.176', 'args': (), 'state': 'running', 'program': '12tb_dsk3/SEIR_model/src/SEIR_skane_kronoberg_sweep_2.ipynb', 'codePathLocal': None, 'root': '/12tb_dsk3/SEIR_model', 'git': {'remote': 'https://github.com/danish2562022/SEIR_model.git', 'commit': '815b8b500e481e3833db0d86f1dc04c90b7aa4bc'}, 'email': 'danish2562022@gmail.com', 'host': 'polster', 'username': 'anwer', 'executable': '/root/miniconda3/bin/python', 'cpu_count': 16, 'cpu_count_logical': 32, 'cpu_freq': {'current': 2582.6304062500003, 'min': 545.0, 'max': 5371.0}, 'cpu_freq_per_core': [{'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 4773.604, 'min': 545.0, 'max': 5371.0}, {'current': 3161.613, 'min': 545.0, 'max': 5371.0}, {'current': 3071.877, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 3333.462, 'min': 545.0, 'max': 5371.0}, {'current': 1983.585, 'min': 545.0, 'max': 5371.0}, {'current': 4713.361, 'min': 545.0, 'max': 5371.0}, {'current': 3382.64, 'min': 545.0, 'max': 5371.0}, {'current': 3124.78, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 2711.079, 'min': 545.0, 'max': 5371.0}, {'current': 4112.273, 'min': 545.0, 'max': 5371.0}, {'current': 4741.119, 'min': 545.0, 'max': 5371.0}, {'current': 4792.036, 'min': 545.0, 'max': 5371.0}, {'current': 4769.019, 'min': 545.0, 'max': 5371.0}, {'current': 4774.003, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 4773.625, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 1947.658, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 4753.958, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 545.0, 'min': 545.0, 'max': 5371.0}, {'current': 3660.262, 'min': 545.0, 'max': 5371.0}], 'disk': {'/': {'total': 3752.004409790039, 'used': 2783.801280975342}}, 'gpu': 'NVIDIA RTX 6000 Ada Generation', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA RTX 6000 Ada Generation', 'memory_total': 51527024640}], 'memory': {'total': 251.08872985839844}}
2025-05-29 21:08:40,849 INFO    HandlerThread:2864158 [system_monitor.py:probe():224] Finished collecting system info
2025-05-29 21:08:40,849 INFO    HandlerThread:2864158 [system_monitor.py:probe():227] Publishing system info
2025-05-29 21:08:40,849 DEBUG   HandlerThread:2864158 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2025-05-29 21:08:41,414 DEBUG   HandlerThread:2864158 [system_info.py:_save_conda():224] Saving conda packages done
2025-05-29 21:08:41,414 INFO    HandlerThread:2864158 [system_monitor.py:probe():229] Finished publishing system info
2025-05-29 21:08:41,417 DEBUG   SenderThread:2864158 [sender.py:send():379] send: files
2025-05-29 21:08:41,417 INFO    SenderThread:2864158 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2025-05-29 21:08:41,596 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: python_packages
2025-05-29 21:08:41,596 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: python_packages
2025-05-29 21:08:41,597 DEBUG   SenderThread:2864158 [sender.py:send():379] send: telemetry
2025-05-29 21:08:41,637 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: internal_messages
2025-05-29 21:08:41,638 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: stop_status
2025-05-29 21:08:41,638 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: stop_status
2025-05-29 21:08:41,645 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: partial_history
2025-05-29 21:08:41,685 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: server_info
2025-05-29 21:08:41,745 INFO    Thread-12 :2864158 [dir_watcher.py:_on_file_created():271] file/dir created: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/wandb-metadata.json
2025-05-29 21:08:41,746 INFO    Thread-12 :2864158 [dir_watcher.py:_on_file_created():271] file/dir created: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/conda-environment.yaml
2025-05-29 21:08:41,746 INFO    Thread-12 :2864158 [dir_watcher.py:_on_file_created():271] file/dir created: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/requirements.txt
2025-05-29 21:08:46,646 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:08:51,646 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:08:56,647 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:01,648 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:06,648 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:11,649 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:16,650 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:21,119 WARNING FileStreamThread:2864158 [file_stream.py:request_with_retry():674] requests_with_retry encountered retryable exception: 500 Server Error: Internal Server Error for url: https://api.wandb.ai/files/danish2562022/sweeps-SEIR_Sk%C3%A5ne_Kronoberg/xb0u8kog/file_stream. func: <bound method Session.post of <requests.sessions.Session object at 0x7c3e4b963ce0>>, args: ('https://api.wandb.ai/files/danish2562022/sweeps-SEIR_Sk√•ne_Kronoberg/xb0u8kog/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2025-05-29 21:09:21,650 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:22,790 INFO    wandb-upload_0:2864158 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 467, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 462, in _make_request
    httplib_response = conn.getresponse()
                       ^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/http/client.py", line 1423, in getresponse
    response.begin()
  File "/root/miniconda3/lib/python3.12/http/client.py", line 331, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/http/client.py", line 292, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/socket.py", line 707, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/ssl.py", line 1252, in recv_into
    return self.read(nbytes, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/ssl.py", line 1104, in read
    return self._sslobj.read(len, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TimeoutError: The read operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 801, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/util/retry.py", line 552, in increment
    raise six.reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/packages/six.py", line 770, in reraise
    raise value
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 469, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 358, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=20)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 345, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/sdk/lib/gql_request.py", line 58, in execute
    request = self.session.post(self.url, **post_args)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/adapters.py", line 713, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=20)
2025-05-29 21:09:22,863 INFO    SenderThread:2864158 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 467, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 462, in _make_request
    httplib_response = conn.getresponse()
                       ^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/http/client.py", line 1423, in getresponse
    response.begin()
  File "/root/miniconda3/lib/python3.12/http/client.py", line 331, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/http/client.py", line 292, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/socket.py", line 707, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/ssl.py", line 1252, in recv_into
    return self.read(nbytes, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/lib/python3.12/ssl.py", line 1104, in read
    return self._sslobj.read(len, buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TimeoutError: The read operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 801, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/util/retry.py", line 552, in increment
    raise six.reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/packages/six.py", line 770, in reraise
    raise value
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 715, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 469, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/home/anwer/.local/lib/python3.12/site-packages/urllib3/connectionpool.py", line 358, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=20)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 345, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/wandb/sdk/lib/gql_request.py", line 58, in execute
    request = self.session.post(self.url, **post_args)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/anwer/.local/lib/python3.12/site-packages/requests/adapters.py", line 713, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=20)
2025-05-29 21:09:26,651 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:31,652 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:34,799 INFO    wandb-upload_0:2864158 [upload_job.py:push():130] Uploaded file /tmp/tmpso3qbc04wandb/wkn5leuy-wandb-metadata.json
2025-05-29 21:09:36,652 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:39,071 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: status_report
2025-05-29 21:09:40,837 DEBUG   SystemMonitor:2864158 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-05-29 21:09:41,095 DEBUG   SenderThread:2864158 [sender.py:send():379] send: history
2025-05-29 21:09:41,095 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: summary_record
2025-05-29 21:09:41,096 INFO    SenderThread:2864158 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-05-29 21:09:41,096 DEBUG   SenderThread:2864158 [sender.py:send():379] send: telemetry
2025-05-29 21:09:41,096 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: server_info
2025-05-29 21:09:41,653 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: keepalive
2025-05-29 21:09:41,759 INFO    Thread-12 :2864158 [dir_watcher.py:_on_file_modified():288] file/dir modified: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/config.yaml
2025-05-29 21:09:41,759 INFO    Thread-12 :2864158 [dir_watcher.py:_on_file_created():271] file/dir created: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/wandb-summary.json
2025-05-29 21:09:42,683 DEBUG   SenderThread:2864158 [sender.py:send():379] send: stats
2025-05-29 21:09:42,685 DEBUG   SenderThread:2864158 [sender.py:send():379] send: exit
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:send_exit():586] handling exit code: 0
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:send_exit():588] handling runtime: 61
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:send_exit():594] send defer
2025-05-29 21:09:42,685 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:42,685 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 0
2025-05-29 21:09:42,685 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 0
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 1
2025-05-29 21:09:42,685 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:42,685 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 1
2025-05-29 21:09:42,685 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 1
2025-05-29 21:09:42,685 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 2
2025-05-29 21:09:42,686 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:42,686 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 2
2025-05-29 21:09:42,686 INFO    HandlerThread:2864158 [system_monitor.py:finish():203] Stopping system monitor
2025-05-29 21:09:42,686 DEBUG   SystemMonitor:2864158 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-05-29 21:09:42,686 DEBUG   SystemMonitor:2864158 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-05-29 21:09:42,686 INFO    HandlerThread:2864158 [interfaces.py:finish():200] Joined cpu monitor
2025-05-29 21:09:42,686 INFO    HandlerThread:2864158 [interfaces.py:finish():200] Joined disk monitor
2025-05-29 21:09:42,727 INFO    HandlerThread:2864158 [interfaces.py:finish():200] Joined gpu monitor
2025-05-29 21:09:42,727 INFO    HandlerThread:2864158 [interfaces.py:finish():200] Joined memory monitor
2025-05-29 21:09:42,728 INFO    HandlerThread:2864158 [interfaces.py:finish():200] Joined network monitor
2025-05-29 21:09:42,728 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:42,728 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 2
2025-05-29 21:09:42,728 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 3
2025-05-29 21:09:42,728 DEBUG   SenderThread:2864158 [sender.py:send():379] send: stats
2025-05-29 21:09:42,728 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:42,728 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 3
2025-05-29 21:09:42,728 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:42,728 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 3
2025-05-29 21:09:42,728 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 4
2025-05-29 21:09:42,728 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:42,728 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 4
2025-05-29 21:09:42,728 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:42,728 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 4
2025-05-29 21:09:42,728 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 5
2025-05-29 21:09:42,729 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:42,729 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 5
2025-05-29 21:09:42,729 DEBUG   SenderThread:2864158 [sender.py:send():379] send: summary
2025-05-29 21:09:42,729 INFO    SenderThread:2864158 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2025-05-29 21:09:42,729 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:42,729 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 5
2025-05-29 21:09:42,729 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 6
2025-05-29 21:09:42,729 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:42,729 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 6
2025-05-29 21:09:42,729 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:42,729 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 6
2025-05-29 21:09:42,730 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: status_report
2025-05-29 21:09:42,759 INFO    Thread-12 :2864158 [dir_watcher.py:_on_file_modified():288] file/dir modified: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/wandb-summary.json
2025-05-29 21:09:43,685 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: poll_exit
2025-05-29 21:09:46,369 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 7
2025-05-29 21:09:46,370 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: poll_exit
2025-05-29 21:09:46,370 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:46,370 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 7
2025-05-29 21:09:46,370 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:46,370 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 7
2025-05-29 21:09:46,370 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 8
2025-05-29 21:09:46,370 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:46,370 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 8
2025-05-29 21:09:46,370 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:46,371 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 8
2025-05-29 21:09:46,371 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 9
2025-05-29 21:09:46,371 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:46,371 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 9
2025-05-29 21:09:46,371 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:46,371 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 9
2025-05-29 21:09:46,371 INFO    SenderThread:2864158 [dir_watcher.py:finish():358] shutting down directory watcher
2025-05-29 21:09:46,685 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: poll_exit
2025-05-29 21:09:46,760 INFO    SenderThread:2864158 [dir_watcher.py:_on_file_modified():288] file/dir modified: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/config.yaml
2025-05-29 21:09:46,761 INFO    SenderThread:2864158 [dir_watcher.py:finish():388] scan: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files
2025-05-29 21:09:46,761 INFO    SenderThread:2864158 [dir_watcher.py:finish():402] scan save: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/conda-environment.yaml conda-environment.yaml
2025-05-29 21:09:46,761 INFO    SenderThread:2864158 [dir_watcher.py:finish():402] scan save: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/config.yaml config.yaml
2025-05-29 21:09:46,761 INFO    SenderThread:2864158 [dir_watcher.py:finish():402] scan save: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/wandb-metadata.json wandb-metadata.json
2025-05-29 21:09:46,761 INFO    SenderThread:2864158 [dir_watcher.py:finish():402] scan save: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/requirements.txt requirements.txt
2025-05-29 21:09:46,763 INFO    SenderThread:2864158 [dir_watcher.py:finish():402] scan save: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/wandb-summary.json wandb-summary.json
2025-05-29 21:09:46,763 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 10
2025-05-29 21:09:46,763 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: poll_exit
2025-05-29 21:09:46,764 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:46,765 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 10
2025-05-29 21:09:46,765 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:46,765 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 10
2025-05-29 21:09:46,765 INFO    SenderThread:2864158 [file_pusher.py:finish():169] shutting down file pusher
2025-05-29 21:09:47,217 INFO    wandb-upload_0:2864158 [upload_job.py:push():130] Uploaded file /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/conda-environment.yaml
2025-05-29 21:09:47,457 INFO    wandb-upload_3:2864158 [upload_job.py:push():130] Uploaded file /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/wandb-summary.json
2025-05-29 21:09:47,493 INFO    wandb-upload_2:2864158 [upload_job.py:push():130] Uploaded file /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/requirements.txt
2025-05-29 21:09:47,661 INFO    wandb-upload_1:2864158 [upload_job.py:push():130] Uploaded file /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/files/config.yaml
2025-05-29 21:09:47,687 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: poll_exit
2025-05-29 21:09:47,687 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: poll_exit
2025-05-29 21:09:47,862 INFO    Thread-11 (_thread_body):2864158 [sender.py:transition_state():614] send defer: 11
2025-05-29 21:09:47,862 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:47,862 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 11
2025-05-29 21:09:47,862 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:47,862 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 11
2025-05-29 21:09:47,863 INFO    SenderThread:2864158 [file_pusher.py:join():175] waiting for file pusher
2025-05-29 21:09:47,863 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 12
2025-05-29 21:09:47,863 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:47,863 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 12
2025-05-29 21:09:47,863 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:47,863 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 12
2025-05-29 21:09:47,863 INFO    SenderThread:2864158 [file_stream.py:finish():601] file stream finish called
2025-05-29 21:09:48,687 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: poll_exit
2025-05-29 21:09:49,440 INFO    SenderThread:2864158 [file_stream.py:finish():605] file stream finish is done
2025-05-29 21:09:49,440 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 13
2025-05-29 21:09:49,440 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: poll_exit
2025-05-29 21:09:49,440 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:49,440 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 13
2025-05-29 21:09:49,441 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:49,441 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 13
2025-05-29 21:09:49,441 INFO    SenderThread:2864158 [sender.py:transition_state():614] send defer: 14
2025-05-29 21:09:49,441 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: defer
2025-05-29 21:09:49,441 INFO    HandlerThread:2864158 [handler.py:handle_request_defer():184] handle defer: 14
2025-05-29 21:09:49,441 DEBUG   SenderThread:2864158 [sender.py:send():379] send: final
2025-05-29 21:09:49,441 DEBUG   SenderThread:2864158 [sender.py:send():379] send: footer
2025-05-29 21:09:49,441 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: defer
2025-05-29 21:09:49,441 INFO    SenderThread:2864158 [sender.py:send_request_defer():610] handle sender defer: 14
2025-05-29 21:09:49,442 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: poll_exit
2025-05-29 21:09:49,442 DEBUG   SenderThread:2864158 [sender.py:send_request():406] send_request: poll_exit
2025-05-29 21:09:49,443 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: internal_messages
2025-05-29 21:09:49,443 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: get_summary
2025-05-29 21:09:49,443 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: sampled_history
2025-05-29 21:09:49,443 DEBUG   HandlerThread:2864158 [handler.py:handle_request():158] handle_request: shutdown
2025-05-29 21:09:49,443 INFO    HandlerThread:2864158 [handler.py:finish():882] shutting down handler
2025-05-29 21:09:50,442 INFO    WriterThread:2864158 [datastore.py:close():296] close: /12tb_dsk3/SEIR_model/src/wandb/run-20250529_210840-xb0u8kog/run-xb0u8kog.wandb
2025-05-29 21:09:50,442 INFO    SenderThread:2864158 [sender.py:finish():1608] shutting down sender
2025-05-29 21:09:50,442 INFO    SenderThread:2864158 [file_pusher.py:finish():169] shutting down file pusher
2025-05-29 21:09:50,442 INFO    SenderThread:2864158 [file_pusher.py:join():175] waiting for file pusher
